Galaxus.Functional/(Result)/ResultExtensions.cs

```diff
@@ -31,7 +31,8 @@ public static class ResultExtensions
         /// <typeparam name="TOkFrom">The current type of <b>Ok</b>. This type must derive from <typeparamref name="TOkTo"/>.</typeparam>
         /// <typeparam name="TOkTo">The type to map <b>Ok</b> to.</typeparam>
         /// <typeparam name="TErr">The type of <b>Err</b> which will remain untouched.</typeparam>
-        public static Result<TOkTo, TErr> Map<TOkFrom, TOkTo, TErr>(this Result<TOkFrom, TErr> self) where TOkFrom : TOkTo
+        public static Result<TOkTo, TErr> Map<TOkFrom, TOkTo, TErr>(this Result<TOkFrom, TErr> self)
+            where TOkFrom : TOkTo
```

Comment:
Yeah, autoformatting I guess. We definitely need to have at least a style guide or something...

The where-clauses are alright, I like them better when they're clearly visible, but there's parameter wrapping for SelectOk which is... "not ok" :D

Reply:
Ok I see... I think a style guide or something would not be bad..